jobbergate-docs/src/core.rst

```diff
@@ -0,0 +1,24 @@
+=================
+ Jobbergate Core
+=================
+
```

Comment:
I feel a little bit of lack of explanation here. See the image below:
![image](https://user-images.githubusercontent.com/45343359/222153020-bcac1351-4d09-490a-a1cc-db601211969e.png)

To understand a little bit better what is the jobbergate-core I need to go to the source code and read the readme. If this is the flow, I think the doc has failed. Maybe a simple inception here improves the doc :)

Reply:
Fair point, I'll elaborate

Reply:
Have you built it locally to review? Thanks for this level of attention to detail.

Reply:
> Have you built it locally to review? Thanks for this level of attention to detail.

I did. Looks great btw. I liked how sphinx used the docstrings to generate the documentation. Do you know if there are ways to create symlinks in the docstrings that sphinx will identify?

Reply:
It is possible, see it produced a link for this cross reference: https://github.com/omnivector-solutions/jobbergate/blob/6e41dd86cf215188a947c25d1e5c579597d8f079/jobbergate-core/jobbergate_core/auth/core.py#L33

Reply:
More details were added on [e7ebf7b](https://github.com/omnivector-solutions/jobbergate/pull/228/commits/e7ebf7b7e4ce8687c48b89c759ea255e98e7e58c)